File: app/src/main/java/com/constraint/vagabond/main/details/SingleImageView.java
package com.constraint.vagabond.main.details;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.constraint.vagabond.R;
import com.github.chrisbanes.photoview.PhotoView;
import com.squareup.picasso.Picasso;

public class SingleImageView extends AppCompatActivity {
  PhotoView photoView;

  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.single_image_view);
    photoView = findViewById(R.id.single_image);
    String imageUrl = getIntent().getStringExtra("image_url");
    Picasso.get().load(imageUrl).into(photoView);
  }
}

File: app/src/main/java/com/sierra/vagabond/main/details/SingleImageView.kt
package com.sierra.vagabond.main.details

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.sierra.vagabond.R
import com.squareup.picasso.Picasso
import kotlinx.android.synthetic.main.single_image_view.*


class SingleImageView : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.single_image_view)
        val imageUrl = intent.getStringExtra("image_url")
        Picasso.get().load(imageUrl).into(single_image)
    }
}
